tests/Fixer/CastNotation/CastSpacesFixerTest.php

```diff
@@ -24,40 +24,50 @@
  */
 final class CastSpacesFixerTest extends AbstractFixerTestCase
 {
-    public function testInvalidConfigMissingKey(): void
+    /**
+     * @param array<string, int|string> $config
```

Comment:
non-blocking

in most places, we say `array<string, mixed>` for config, as config under given option key can also be array/assoc-array.

maybe this single rule is indeed accepting only int|string

Reply:
It is indeed that type, I wouldn't care much about the types for now, as I imagine we have the `testFix*` methods in parent class when we have the names aligned eveerywhere.